platformio.ini

```diff
@@ -280,6 +280,19 @@ lib_ignore    = Adafruit NeoPixel, c1921b4, TMCStepper
 src_filter    = ${common.default_src_filter} +<src/HAL/HAL_STM32F4>
 monitor_speed = 250000
 
+#
+# ARMED
+#
+[env:ARMED]
+platform      = ststm32
+framework     = arduino
+board         = ARMED
+build_flags   = ${common.build_flags} -DUSE_STM32GENERIC -DSTM32GENERIC -DMENU_USB_SERIAL -DMENU_SERIAL=SerialUSB -O2 -ffreestanding -fsigned-char -fno-move-loop-invariants -fno-strict-aliasing -std=gnu11 -std=gnu++11
+lib_deps      = ${common.lib_deps}
+lib_ignore    = Adafruit NeoPixel, c1921b4
+src_filter    = ${common.default_src_filter} +<src/HAL/HAL_STM32F4>
+monitor_speed = 250000
+
```

Comment:
This looks like it's needed. But it makes us worry that more boards will be coming along that need their own env entries. It would be nice if we could get more to fit under the more generic environments.

Reply:
It's actually only needed to compile with the STM32Generic core. When the official STM32 Arduino Core is integrated with PlatformIO it might be possible to use a more generic approach.

Thanks for the merge.